BUILD

```diff
@@ -53,171 +59,147 @@ filegroup(
         "test/io.tcl",
         ":util",
     ],
-    visibility = [":__subpackages__"],
 )
 
 filegroup(
     name = "constraints-sram",
     srcs = [
         "test/constraints-sram.sdc",
+    ],
+    data = [
         ":util",
     ],
     visibility = [":__subpackages__"],
 )
 
-build_openroad(
-    name = "tag_array_64x184",
-    abstract_stage = "floorplan",
-    io_constraints = ":io-sram",
-    sdc_constraints = ":constraints-sram",
-    stage_args = {
-        "floorplan": [
-            "CORE_UTILIZATION=40",
-            "CORE_ASPECT_RATIO=2",
-        ],
-        "place": ["PLACE_DENSITY=0.65"],
+synth(
+    name = "tag_array_synth",
+    arguments = {
+        "SDC_FILE": "$(location :constraints-sram)",
     },
-    verilog_files = ["//another:tag_array_64x184.sv"],
-    visibility = [":__subpackages__"],
+    data = [
+        ":constraints-sram",
+    ],
+    module_top = "tag_array_64x184",
+    verilog_files = ["test/mock/tag_array_64x184.sv"],
+)
+
+SRAM_FLOOR_PLACE_ARGUMENTS = {
+    "IO_CONSTRAINTS": "$(location :io-sram)",
+}
+
+floorplan(
+    name = "tag_array_floor",
```

Comment:
I don't like that we have to invent names here for each stage. The user now has to come up with lots and lots of names and even within this example, it is done inconsistently.

I don't have a suggestion on how to address this.

Reply:
I agree that it's verbose, but how are you supposed to refer to the output of a stage without giving it a name?